cost/cloud_cost_anomaly_alerts/README.md

```diff
@@ -23,6 +23,8 @@ Any point outside of the bands is considered as anomalous. If multiple cost anom
   - Default: 1000
 - *Cost Anomaly Dimensions* - Dimensions to group data for the analysis. Multiple dimensions can be provided. Currently, custom tags cannot be used as dimensions.
   - Default: ["Cloud Vendor Account", "Cloud Vendor", "Service"]
+- *Rule Based Dimensions* - Rule Based Dimensions to group data for the analysis. Multiple rbd id's or name's can be provided. This allows for better visualization of data that was not tagged or difficult to tag in the native cloud platform, these new dimensions in Optima will be populated according to rules you define, you can use the new dimensions to look at your cost from new meaningful perspectives, a common use case is to create several dimensions like CCO 'Environments' or 'Product Line', populated based on know account values.
+- *User dimensions* - User Rule Based Dimensions to group data for the analysis. Multiple rbd id's or name's can be provided.
```

Comment:
here also, In user dimension you mentioned Rule based dimension, that is not right.

Rule based dimensions are different from standard dimension can you remove "rule"